bin/publish

```diff
@@ -5,7 +5,7 @@
 REDHAT_IMAGE='scan.connect.redhat.com/ospid-1c46a2de-1d88-40e6-a433-7114ad0099cb/conjur-openshift-authenticator-client'
 
 readonly REGISTRY="cyberark"
-readonly VERSION="$(short_version_tag)"
+readonly VERSION="${TAG_NAME//"v"}"
```

Comment:
can you explain this please?

Reply:
We want to align this repo with other pipeline's standards - using the TAG_NAME variable which has the version in its value - just removing the "v" prefix in order to have the number's version

Reply:
can you add a comment saying that we take the `TAG_NAME` variable and remove the `v` prefix? i didn't know what this syntax is